### DISCUSSION TITLE
Addition to README

### DISCUSSION THREAD

**Discussion by rleaver152** [policratus/pupyl#126] [Show and tell]
If you are on a Mac like me,   one of the components of Pupyl is annoy==1.17.0     However if you try to install it and have to build the wheel,  with a brew installed gcc ,  then it will fail.     The solution is to use the installed Clang compiler by simply issuing 

export CC=/usr/bin/clang
export CXX=/usr/bin/clang++

before doing 

pip install pupyl 

This may save some newer users a world of pain as its not really documented on Prof Google.

**Comment by policratus** [comment]
Hi @rleaver152, thanks for that!

Actually on [`README.md`](https://github.com/policratus/pupyl/blob/main/README.md), [__Installation__](https://github.com/policratus/pupyl/blob/main/README.md#-installation) section, there's a link to the [`TROUBLESHOOTING.md`](https://github.com/policratus/pupyl/blob/main/TROUBLESHOOTING.md), which is the place to help users on the installation process.

Adding a section on [`TROUBLESHOOTING.md`](https://github.com/policratus/pupyl/blob/main/TROUBLESHOOTING.md) regarding Mac users is great because there's nothing there.

Just opened #130 adding this. @rleaver152, could help reviewing that? I'm not a macOS user, so it's harder for me to verify.

**Comment by policratus** [comment]
@rleaver152, just merged the pull request #130. Thanks a lot for your contribution!
